### DISCUSSION TITLE
How to show top menu bar when reading a book?

### DISCUSSION THREAD

**Discussion by homocomputeris** [arianneorpilla/jidoujisho#292] [Q&A] [answered]
How to shop top menu bar when reading a book? The contents button is only there.

**Comment by arianneorpilla** [comment]
Refer to the [ttu readme](https://github.com/ttu-ttu/ebook-reader)  for documentation

**Comment by homocomputeris** [comment, marked as answer]
Tap on the top of the screen where the menu bar should be. Swiping down won't work.
